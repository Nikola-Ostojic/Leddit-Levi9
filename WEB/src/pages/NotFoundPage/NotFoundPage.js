import React from "react";
import { Container } from "@material-ui/core";
import { withRouter } from "react-router-dom";

export const NotFoundPage = ({ location: { pathname } }) => (
  <Container>Page not found url: {pathname}</Container>
);

export default withRouter(NotFoundPage);
