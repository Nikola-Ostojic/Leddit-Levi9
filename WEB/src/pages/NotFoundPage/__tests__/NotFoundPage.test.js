import React from "react";
import { shallow } from "enzyme";

import { NotFoundPage } from "../NotFoundPage";

describe("NotFoundPage component tests", () => {
  it("Should match snapshot", () => {
    const mockedProps = {
      location: {
        pathname: "Not existing route"
      }
    };
    const wrapper = shallow(<NotFoundPage {...mockedProps} />);

    expect(wrapper).toMatchSnapshot();
  });
});
