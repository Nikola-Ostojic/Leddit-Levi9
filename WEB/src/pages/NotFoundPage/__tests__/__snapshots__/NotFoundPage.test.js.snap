// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage component tests Should match snapshot 1`] = `
<WithStyles(ForwardRef(Container))>
  Page not found url: 
  Not existing route
</WithStyles(ForwardRef(Container))>
`;
